Tests/DependencyInjection/Compiler/AddVotersPassTest.php

```diff
@@ -27,15 +28,63 @@ public function testProcessWithAlias()
         $definitionMock = $this->getMockBuilder('Symfony\Component\DependencyInjection\Definition')
             ->disableOriginalConstructor()
             ->getMock();
-        $definitionMock->expects($this->at(0))
-            ->method('addMethodCall')
-            ->with($this->equalTo('addVoter'), $this->equalTo(array(new Reference('id'))));
-        $definitionMock->expects($this->at(1))
-            ->method('addMethodCall')
-            ->with($this->equalTo('addVoter'), $this->equalTo(array(new Reference('foo'))));
-        $definitionMock->expects($this->at(2))
-            ->method('addMethodCall')
-            ->with($this->equalTo('addVoter'), $this->equalTo(array(new Reference('bar'))));
+
+        $voters = array(new Reference('id'), new Reference('foo'), new Reference('bar'));
+
+        if (class_exists(IteratorArgument::class)) {
+            $voters = new IteratorArgument($voters);
+        }
+
+        $definitionMock->expects($this->once())
+            ->method('replaceArgument')
+            ->with(0, $voters);
+
+        $listenerMock = $this->getMockBuilder('Symfony\Component\DependencyInjection\Definition')
```

Comment:
i would use the ::class pseudo constant for these. and can't you do `createMock` instead of getMockBuilder->disableOriginalConstructor()->getMock()? with the phpunit bridge we should have a recent enough version of phpunit i think.

Reply:
Well, the argument for using a recent version of PHPUnit is our drop of PHP 5.5 actually. Even when using the bridge, you would get PHPUnit 4.8 when running on PHP 5.5 (as PHPUnit 5 requires PHP 5.6+)